package.json

```diff
@@ -67,11 +67,11 @@
     "@uniswap/token-lists": "^1.0.0-beta.30",
     "@uniswap/v2-sdk": "^3.0.1",
     "@uniswap/v3-sdk": "^3.8.2",
-    "@web3-react/core": "8.0.30-beta.0",
-    "@web3-react/eip1193": "8.0.23-beta.0",
-    "@web3-react/empty": "8.0.17-beta.0",
-    "@web3-react/types": "8.0.17-beta.0",
-    "@web3-react/url": "8.0.22-beta.0",
+    "@web3-react/core" : "8.0.35-beta.0",
+    "@web3-react/eip1193": "8.0.26-beta.0",
+    "@web3-react/empty": "8.0.20-beta.0",
+    "@web3-react/types": "8.0.20-beta.0",
+    "@web3-react/url": "8.0.25-beta.0",
```

Comment:
@vm confirming these patch versions are the correct ones we should release with

Reply:
whatever is latest is what i'd use